MYabrv.bib

```diff
@@ -204,6 +203,7 @@ @String{WEA
 
 symposium
 @String{DLS = "Proc.\ Symposium on Dynamic Languages (DLS)"}
+@String{ESEM = "Proc.\ of the Int'l Conf.\ Symposium on Empirical Software Engineering and Measurement (ESEM)"}
```

Comment:
Sorry, if this was my fault, but `Conf.\` is too much here because it is not a conference.

Reply:
Oh yes this was my fault, sorry! :(